js/letters.js

```diff
@@ -6,20 +6,28 @@ cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non \
 proident, sunt in culpa qui officia deserunt mollit anim id est laborum.";
 
 var counter = {
-  "a" : 0,  "b" : 0,  "c" : 0,  "d" : 0,
-  "e" : 0,  "f" : 0,  "g" : 0,  "h" : 0,
-  "i" : 0,  "j" : 0,  "k" : 0,  "l" : 0,
-  "m" : 0,  "n" : 0,  "o" : 0,  "p" : 0,
-  "q" : 0,  "r" : 0,  "s" : 0,  "t" : 0,
-  "u" : 0,  "v" : 0,  "w" : 0,  "x" : 0,
-  "y" : 0,  "z" : 0
+  "a": 0, "b": 0, "c": 0, "d": 0,
+  "e": 0, "f": 0, "g": 0, "h": 0,
+  "i": 0, "j": 0, "k": 0, "l": 0,
+  "m": 0, "n": 0, "o": 0, "p": 0,
+  "q": 0, "r": 0, "s": 0, "t": 0,
+  "u": 0, "v": 0, "w": 0, "x": 0,
+  "y": 0, "z": 0
 }
 
-function countLetters(counter, sample_text){
-  // FIX ME
-}
+function countLetters(counter, sample_text) {
+  let text = sample_text.toLowerCase();
+  if (sample_text.length === 0) {
+    return counter;
+  }
+  let letter = text.slice(0, 1);
+  if ([letter]) {
```

Comment:
The square brackets here are not necessary and actually makes the if statement true at all times. 

You'll also need to figure out if the particular character you're getting is in the counter object or not. Check out the object.hasOwnProperty method for a way to double check if a key exists in an object.